src/clib/pio.h

```diff
@@ -283,7 +283,11 @@
 #endif
 
 /** ??? */
-#define PIO_REQ_NULL (NC_REQ_NULL-1)
+#ifdef _PNETCDF
```

Comment:
The comment ??? from legacy code might be removed.
Also, PIO_REQ_NULL can be defined in the same place
where PIO_EINDEP is defined (line 269 to 273).

Reply:
ok, I will remove the ??? comment and move it near PIO_EINDEP

Reply:
This should be fixed now